### PR TITLE
Refactor the workflow of containers

### LINKED ISSUE #609: Remove workflow limitations with makefile

### PR DESCRIPTION
**Pull request by esloch**
Closes #609.
- Add makim.
  - Improve workflow and use of conditional IFS arguments.
- Add container-sugar.
  - Create service groups.
- Update compose file.

### CODE REVIEW

File: conda/dev.yaml [hunk 1]
```diff
@@ -1,9 +1,12 @@
 name: alertadengue
 channels:
   - conda-forge
-  - nodefaults
```
Comment (xmnlab) [inline conda/dev.yaml:4]: You probably should keep nodefaults to avoid mixing packages from anaconda and conda forge

File: conda/dev.yaml [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 dependencies:
   - gdal 3.3.*
+  - pip
   - poetry
   - psycopg2-binary
   - python 3.9.*
+  - pip:
+    - containers-sugar >=1.1.0
```
Comment (xmnlab) [inline conda/dev.yaml:11]:
```suggestion
    - containers-sugar >=1.2.0
```

**Review by xmnlab — CHANGES_REQUESTED** [review]
The targets on makim should be fixed and added two notes ... it is a good progress!!

**Review by fccoelho — APPROVED** [review]

**Comment by xmnlab** [comment]
is it already working here?

**Comment by esloch** [comment]
I don't know why the wait don't exit to tmate.

**Pull request merged**